tmt/cli.py

```diff
@@ -164,6 +164,10 @@ def main(click_contex, root, context, **kwargs):
         print(f"tmt version: {tmt.__version__}")
         raise SystemExit(0)
 
+    # Disable coloring if NO_COLOR is set
+    if 'NO_COLOR' in os.environ:
```

Comment:
@FrNecas it would be also nice to update the documentation with this environment variable

Reply:
Thanks for the suggestion, didn't know we had a section for environment variables ;) Added.